packages/@react-spectrum/table/src/Nubbin.tsx

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright 2022 Adobe. All rights reserved.
+ * This file is licensed to you under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License. You may obtain a copy
+ * of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under
+ * the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+ * OF ANY KIND, either express or implied. See the License for the specific language
+ * governing permissions and limitations under the License.
+ */
+
+import React from 'react';
+
+
+// TODO resize with scale? colors should be variables
+export function Nubbin() {
+  return (
+    <svg xmlns="http://www.w3.org/2000/svg" width="16" height="16" viewBox="0 0 16 16">
+      <g id="Group_175838" data-name="Group 175838">
+        <g id="Ellipse_10104" data-name="Ellipse 10104" fill="var(--spectrum-global-color-blue-600)" stroke="var(--spectrum-global-color-blue-600)" strokeWidth="2">
+          <circle cx="8" cy="8" r="8" stroke="none" />
+          <circle cx="8" cy="8" r="7" fill="none" />
+        </g>
+      </g>
+      <path id="Path_104166" data-name="Path 104166" d="M-2106-7380.263v5l2.5-2.551Z" transform="translate(2116 7385.763)" fill="#fff" stroke="#fff" strokeLinejoin="round" strokeWidth="2" />
+      <path id="Path_104167" data-name="Path 104167" d="M-2106-7380.263v5l2.5-2.551Z" transform="translate(-2100 -7369.763) rotate(180)" fill="#fff" stroke="#fff" strokeLinejoin="round" strokeWidth="2" />
+    </svg>
```

Comment:
Would the nubbin ever switch its appearance to inform the user that they've reached the min/max of the allowed drag position or are we relying on the mouse cursor to do that? If not, should the nubbin still have arrows?

Reply:
right now, no plan for this, can ask design, but it'd be followup